recipe/recipe.yaml

```
@@ -45,19 +49,21 @@ requirements:
     - python
     - vector
 
-test:
-  imports:
-    - fastjet
-  commands:
-    - pip check
-  requires:
-    - pip
+tests:
+  - python:
+      imports:
+        - fastjet
+  - requirements:
+      run:
+        - pip
+    script:
+      - pip check
```

```suggestion
tests:
  - python:
      imports:
        - fastjet
      pip_check: true
  - requirements:
      run:
        - pip
        - python
    script:
      - pip list
```

We can use the `pip_check` infrastructure now, but it can still be useful to see the `pip list` output for checking everything that `pip` "sees".